docker-compose.yml

```diff
@@ -7,9 +7,12 @@ services:
       - 8080:8080
     devices:
       - /dev/isgx
+    volumes:
+      - /home/romain/Documents/rest_app/demo:/home/romain/Documents/rest_app/demo
```

Comment:
You should avoid using hardcoded paths on (at least) left side.
Best practice is declaring a docker volume (externally declared will help you keeping data between runs, looks kind similar to external docker networks).